src/privacy/analytics.js

```diff
@@ -1,21 +1,24 @@
-import gtag from '../utils/gtag'
+import ReactGA from 'react-ga4'
 import * as Functional from './functional'
 import * as Privacy from './Privacy'
 import {assertDefined} from '../utils/assert'
 
 
+ReactGA.initialize('G-GRLNVMZRGW')
```

Comment:
Does the initialization need to be unconditional?

Can the production value be injected via an environment variable (if present), rather than hard-coded?

Reply:
Actually, I take it back.  I rewrote the privacy system.  The old setup assumed a global GA enable, which was for separate usage vs. social, which we didn't make use of.  Functional cookies are now just set directly in `firstTime` and `preferences`, and `analytics` does a lazy init only if enabled, which is true by default or until the PrivacyControl toggle.